components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -130,16 +130,18 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
         }
 
         try {
-            const watchDispose = watchWorkspaceStatus(this.state.workspace?.id, (resp) => {
-                if (resp.workspaceId !== this.state.workspace?.id || !resp.status) {
+            const watchDispose = watchWorkspaceStatusInOrder(this.props.workspaceId, 99, async (resp) => {
```

Comment:
If we're choosing arbitrary numbers here, we should document them or use something like an ENUM instead with a low and high priority, if those are the only ones we're using.

Reply:
Let's also put a better jsdoc comment on `watchWorkspaceStatusInOrder`, I think it could be very confusing what priorities mean in this context.

Reply:
Feedback addressed